packages/aws-cdk/lib/api/logs/find-cloudwatch-logs.ts

```diff
@@ -0,0 +1,108 @@
+import * as cxapi from '@aws-cdk/cx-api';
+import { Mode, SdkProvider, ISDK } from '../aws-auth';
+import { EvaluateCloudFormationTemplate, LazyListStackResources, establishResourcePhysicalName } from '../evaluate-cloudformation-template';
+
+// resource types that have associated CloudWatch Log Groups that should
+// _not_ be monitored
```

Comment:
No need to be skimpy on the line length here, considering how long the next line is:

```suggestion
// resource types that have associated CloudWatch Log Groups that should _not_ be monitored
```